### PR TITLE
Shown as confirmation after various actions

### PR DESCRIPTION
**Pull request by merad99**

### CODE REVIEW

File: src/components/ConfirmationBox.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Button, Row, Modal } from "react-bootstrap";
+
+import '../styles/ConfirmationBox.css'
+
+function ConfirmationBox(props) {
+    return (
+        <Modal
```
Comment (iamamonte) [inline src/components/ConfirmationBox.js:8]: Let's decouple the confirmation from a modal. It was a reasonable thought, but let's keep them separate.

File: src/components/ConfirmationBox.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            centered>
+            <Modal.Header closeButton></Modal.Header>
+
+            <Modal.Body>
+                <Row className='justify-content-center'><span className="body-modal-icon"><i class="fas fa-check-circle fa-3x"></i></span></Row>
+                <Row className='justify-content-center'><h1 className="body-modal-h1">Post Successfully!</h1></Row>
+                <Row className='justify-content-center'><p className="body-modal-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh</p></Row>
+            </Modal.Body>
+
+            <Row className='justify-content-center'><Button onClick={props.onHide} className="button-modal">OK</Button></Row>
```
Comment (iamamonte) [inline src/components/ConfirmationBox.js:21]: Header and content need to be set by props.

File: src/styles/ConfirmationBox.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.button-modal{
+
+width: 246px;
+height: 77px;
+background: #402366 0% 0% no-repeat padding-box;
+opacity: 1;
+
+}
```
Comment (iamamonte) [inline src/styles/ConfirmationBox.css:40]: Let's avoid hardcoding widths and heights

File: src/components/ConfirmationBox.js (hunk 1 above)
Comment (merad99) [reply src/components/ConfirmationBox.js:8]: ok, maybe just a simple container?

File: src/styles/ConfirmationBox.css (hunk 3 above)
Comment (merad99) [reply src/styles/ConfirmationBox.css:40]: got it. I wasn't sure if the style that came with the app-wireframe should be implemented

Comment (iamamonte) [reply src/styles/ConfirmationBox.css]: See https://github.com/Turnout-Activism/ReactPlayground/pull/49/commits/6a4b7014d88f98edd41208bd2094204d8e7a720c

File: src/components/ConfirmationBox.js (hunk 2 above)
Comment (iamamonte) [reply src/components/ConfirmationBox.js]: See https://github.com/Turnout-Activism/ReactPlayground/pull/49/commits/23952b035e8f8856f6fbc4659d732bf894424889

File: src/components/ConfirmationBox.js (hunk 1 above)
Comment (iamamonte) [reply src/components/ConfirmationBox.js]: See https://github.com/Turnout-Activism/ReactPlayground/pull/49/commits/14ba8e928181379dbe86d89c209d5e5f98539696

**Pull request merged**